src/main/java/com/rabbitmq/perf/MulticastSet.java

```diff
@@ -116,6 +116,10 @@ public void run(boolean announceStartup)
 
         for (Thread consumerThread : consumerThreads) {
             consumerThread.start();
+            if(params.getConsumerSlowStart()) {
+            	System.out.println("slowStart: wait 1sec");
```

Comment:
Can we make it say "Delaying start of next consumer for 1 second because of -S…"?